tests/SpecTests/ClientSideEncryptionSpecTest.php

```diff
@@ -569,76 +557,173 @@ public function testCorpus($schemaMap = true)
             switch ($fieldName) {
                 case '_id':
                 case 'altname_aws':
+                case 'altname_azure':
+                case 'altname_gcp':
                 case 'altname_local':
                     $corpusCopied[$fieldName] = $data;
                     break;
 
                 default:
-                    $corpusCopied[$fieldName] = $this->prepareCorpusData($data, $clientEncryption);
+                    $corpusCopied[$fieldName] = $this->prepareCorpusData($fieldName, $data, $clientEncryption);
             }
         }
 
         $collection->insertOne($corpusCopied);
         $corpusDecrypted = $collection->findOne(['_id' => 'client_side_encryption_corpus']);
 
         $this->assertDocumentsMatch($corpus, $corpusDecrypted);
+
+        $corpusEncryptedExpected = (array) $this->decodeJson(file_get_contents(__DIR__ . '/client-side-encryption/corpus/corpus-encrypted.json'));
+        $corpusEncryptedActual = $client->selectCollection('db', 'coll')->findOne(['_id' => 'client_side_encryption_corpus'], ['typeMap' => ['root' => 'array', 'document' => stdClass::class, 'array' => 'array']]);
+
+        foreach ($corpusEncryptedExpected as $fieldName => $expectedData) {
+            switch ($fieldName) {
+                case '_id':
+                case 'altname_aws':
+                case 'altname_azure':
+                case 'altname_gcp':
+                case 'altname_local':
+                    continue 2;
+            }
```

Comment:
Optional suggestion:

```php
if (in_array($fieldName, ['_id', 'altname_aws', 'altname_azure', 'altname_gcp', 'altname_local'])) {
    continue;
}
```

I forget if there's a sniff that complains about a missing `default:` case (I could be thinking of C).

Reply:
No warnings in PHP, but changed it regardless, including removing the duplicated `switch` from a few lines earlier.